app/controllers/auctions_controller.rb

```diff
@@ -48,28 +45,16 @@ def previous_winners_archive
   end
 
   def previous_winners
-    collection = AuctionQuery.new.public_index
-    @view_model = AuctionCollection.new(current_user, collection)
-
-    @auctions_json = @view_model.auctions.each { |a| AuctionSerializer.new(a, root: false) }.as_json
-
-    respond_to do |format|
-      format.html
-    end
-  end
-
-  rescue_from 'ActiveRecord::RecordNotFound' do
```

Comment:
@adelevie thoughts n this? I myself would default to `ActiveRecord::RecordNotFound` rather than making it a routing error since that is a more obvious error from the dev perspective (saves time debugging) but both would be 404 on prod..